stress/shell/src/main/java/alluxio/stress/cli/MasterBatchTask.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.cli;
+
+import alluxio.stress.Summary;
+import alluxio.stress.master.MasterBatchTaskParameters;
+import alluxio.util.JsonSerializable;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParametersDelegate;
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Multiple tasks runner for Master StressBench.
+ */
+public class MasterBatchTask extends BatchTask {
+  private static final Logger LOG = LoggerFactory.getLogger(MasterBatchTask.class);
+  // Target throughput is set to a huge number to reach the maximum throughput of the task.
+  private static final int TARGET_THROUGHPUT  = 1_000_000;
+  private static final String WARMUP = "0s";
+  // Duration is set to a huge number to ensure that the task is finished
+  private static final String DURATION = "6h";
+  private static final String CLUSTER = "--cluster";
+
+  private String mOperation;
+
+  //store the possible operations
+  static Map<String, String> sSecondOperationMapping = ImmutableMap.<String, String>builder()
```

Comment:
This is an immutable map, so can be made final. Also I think the generic type arguments do not have to be repeated.

```suggestion
  private static final Map<String, String> SECOND_OPERATION_MAPPING = ImmutableMap.builder()
```

Reply:
Since this is a task name -> operation name mapping, how about renaming it to `TASK_OPERATION_MAP`? Also leave a comment on what the keys and values are, respectively.